### PR TITLE
tee loopback documentation

### PR DESCRIPTION
**Pull request by aryszka**
Signed-off-by: Arpad Ryszka <arpad.ryszka@gmail.com>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: docs/tutorials/shadow-traffic.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Shadow Traffic
+
+This tutorial will show how to setup routing for shadow traffic, where one backend (main) will receive the full
+traffic, while a shadowing backend (test) will receive only a certain percentage of the same traffic.
+
+#### Used Predicates:
+
+* [Tee](../../reference/predicates/#tee)
```
Comment (szuecs) [inline docs/tutorials/shadow-traffic.md:8]: IIRC we should use `../../reference/predicates.md#tee`

Comment (szuecs) [reply docs/tutorials/shadow-traffic.md:8]: please also for the other links :)

Comment (aryszka) [reply docs/tutorials/shadow-traffic.md:8]: will check :+1:

Comment (aryszka) [reply docs/tutorials/shadow-traffic.md]: indeed. Both work, but if the links are in the suggested way, then they are also validated. Thx!

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**